src/main/resources/css/reporting.css

```diff
@@ -154,6 +154,12 @@ table.step-arguments {
     margin-top: 3px;
 }
 
+table.step-docstring {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (table.step-docstring) is overqualified, just use .step-docstring without element name.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2669731470/issues/source?bid=3059308&fileBranchId=3332252#l157)